platforms/documentation/docs/src/samples/writing-tasks/custom-test-task/groovy/build.gradle

```diff
@@ -0,0 +1,5 @@
+plugins {
+    id 'com.example.custom-test'
+}
+
+version = '1.0.2'
```

Comment:
💭 This is the tension in documentation we just talked about re: Declarative Gradle.  Adding plugin to define the task is the proper way to do things, yet it makes this example much more complex and distracts from the important parts with a lot of ceremony for setting up the included plugin build.

I'd argue it's not worth having this one example do this if all the others don't.  We should update all examples to do this at once, so this doesn't stand out (wrongly) as far more complicated.  I think it would be better to just define the task in the buildscript directly for now.